src/main/java/org/opentripplanner/transit/model/network/grouppriority/DefaultTransitGroupPriorityCalculator.java

```diff
@@ -10,7 +10,7 @@ public final class DefaultTransitGroupPriorityCalculator
   implements RaptorTransitGroupPriorityCalculator {
 
   @Override
-  public int mergeGroupIds(int currentGroupIds, int boardingGroupId) {
+  public int mergeInGroupId(int currentGroupIds, int boardingGroupId) {
```

Comment:
What does it mean to "merge in a group id" rather than merging two group ids? I would assuming that "merging in" modifies an existing instance.

Reply:
The first argument is a set of group ids, not singular.

Reply:
I guess both names work, but the name should match the JavaDoc:

```
/**
   * Merge in the transit group id with an existing set. Note! Both the set
   * and the group id type is {@code int}.
   *
   * @param currentGroupIds the set of groupIds for all legs in a path.
   * @param boardingGroupId the transit group id to add to the given set.
   * @return the new computed set of groupIds
   */
   ```